src/helpers/runSyncModules.js

```diff
@@ -0,0 +1,29 @@
+import promiseTry from 'promise.try';
+
+import interopRequireDefault from './interopRequireDefault';
+import getModuleFromPath from './getModuleFromPath';
+
+function getSyncModule(config, syncScriptPath) {
+  const {
+    projectRoot = process.cwd(),
+  } = config;
+
+  return getModuleFromPath(syncScriptPath, {
+    projectRoot,
+    requireInteropWrapper: interopRequireDefault,
+  });
+}
+
+export default function runSyncModules(config) {
+  const {
+    sync,
+  } = config;
+
+  return sync.hooks.reduce(
+    (prev, syncScriptPath) => prev.then(() => {
+      const module = getSyncModule(config, syncScriptPath);
+      return promiseTry(() => module(config));
+    }),
+    Promise.resolve(),
+  ).then(() => {});
```

Comment:
can we add a test that fails without this added `.then`?

Reply:
yes https://github.com/airbnb/react-component-variations/pull/15/files#diff-b766422493f4afc8092bf01a3dda7712R30